gdp/small_batch/gdp_small_batch.py

```diff
@@ -9,99 +17,200 @@
 
 
 def build_small_batch():
+    """
+    The function build the GDP model for the small batch problem.
+
+    Args:
+        None
+    
+    Returns:
+        m: pyomo.ConcreteModel
+    """
     NK = 3
 
     # Model
     m = pe.ConcreteModel()
 
     # Sets
-    m.i = pe.Set(initialize=['a', 'b'])  # Set of products
+    m.i = pe.Set(initialize=['a', 'b'], doc='Set of produncts')  # Set of products, i=1,2
     m.j = pe.Set(initialize=['mixer', 'reactor',
-                             'centrifuge'])  # Set of stages
+                             'centrifuge'])  # Set of stages, j=1,2,3
     m.k = pe.RangeSet(NK)    # Set of potential number of parallel units
 
     # Parameters and Scalars
 
-    m.h = pe.Param(initialize=6000)  # Horizon time  (available time hrs)
-    m.vlow = pe.Param(initialize=250)  # Lower bound for size of batch unit
-    m.vupp = pe.Param(initialize=2500)  # Upper bound for size of batch unit
+    m.h = pe.Param(initialize=6000, doc='Horizon time [hr]')  # Horizon time  (available time) [hr]
+    m.vlow = pe.Param(initialize=250, doc='Lower bound for size of batch unit [L]')  # Lower bound for size of batch unit [L]
+    m.vupp = pe.Param(initialize=2500, doc='Upper bound for size of batch unit [L]')  # Upper bound for size of batch unit [L]
 
     # Demand of product i
-    m.q = pe.Param(m.i, initialize={'a': 200000, 'b': 150000})
+    m.q = pe.Param(m.i, initialize={'a': 200000, 'b': 150000}, doc='Production rate pf the product [kg/]')
     # Cost coefficient for batch units
     m.alpha = pe.Param(
-        m.j, initialize={'mixer': 250, 'reactor': 500, 'centrifuge': 340})
+        m.j, initialize={'mixer': 250, 'reactor': 500, 'centrifuge': 340}, doc='Cost coefficient for batch units')
     # Cost exponent for batch units
     m.beta = pe.Param(
-        m.j, initialize={'mixer': 0.6, 'reactor': 0.6, 'centrifuge': 0.6})
+        m.j, initialize={'mixer': 0.6, 'reactor': 0.6, 'centrifuge': 0.6}, doc='Cost exponent for batch units')
```

Comment:
Units?

Reply:
It is exponent on the volume [L]. I don't think adding the unit on the exponent works.